app/views/local_authorities/_filter.html.erb

```diff
@@ -0,0 +1,45 @@
+<div class="app-view-filter govuk-!-margin-right-5 govuk-!-padding-5">
```

Comment:
I think we could remove the `govuk-!-margin-right-5` class to give a little more space.

Reply:
Not using the app-view-filter block any more.